regression/benchmark_scripts/benchmark.py

```
@@ -0,0 +1,346 @@
+
+    updated_data.to_csv(file_path)
+
+
+def run_benchmark(
+    test_name,
+    data_model,
+    solver_settings,
+    required_metrics,
+    csv_path,
+    git_commit,
+    test_status_file,
```

Ruff (0.14.2)

Unused function argument: `test_status_file`

(ARG001)